types/taiko/test/selectors.ts

```diff
@@ -173,3 +173,7 @@ text('Vehicle'); // $ExpectType TextWrapper
 text('Vehicle', below('text')); // $ExpectType TextWrapper
 text('Vehicle', { exactMatch: true }, below('text')); // $ExpectType TextWrapper
 text('Vehicle', { selectHiddenElements: true }); // $ExpectType TextWrapper
+text('Vehicle', { selectHiddenElements: true }); // $ExpectType TextWrapper
+text('/Vehicle/'); // $ExpectType TextWrapper
```

Comment:
I don't think regex as string should be supported. This can be a valid text which will then be considered a regex leading to unwanted matches and escaping. I think it is ok to explicitly use `/test/` or `new Regexp` to indicated a search by regex.

Reply:
It is actually the side effect of serializing object when running on browser to find  regex matches, even the Regex object is converted to string. Just wanted to explicitly call out so that users can know how a string starting and ending with a `/` is treated